From adda763c2dc4eddfb84c7294f92a187b697df947 Mon Sep 17 00:00:00 2001
From: "Dr. David Alan Gilbert" <dgilbert@redhat.com>
Date: Thu, 30 Jul 2015 12:07:55 +0200
Subject: [PATCH 04/13] Remove unneeded memset

Message-id: <1438258084-15753-5-git-send-email-dgilbert@redhat.com>
Patchwork-id: 67210
O-Subject: [RHEL-7.1.Z qemu-kvm-rhev PATCH 04/13] Remove unneeded memset
Bugzilla: 1248382
RH-Acked-by: Marcel Apfelbaum <marcel@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>

From: "Dr. David Alan Gilbert" <dgilbert@redhat.com>

Signed-off-by: Dr. David Alan Gilbert <dgilbert@redhat.com>
Reviewed-by: Michael R. Hines <mrhines@us.ibm.com>
Signed-off-by: Juan Quintela <quintela@redhat.com>
(cherry picked from commit 4fa3dd17dc29c316726f0d4a354a4d895e130c73)
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 migration-rdma.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/migration-rdma.c b/migration-rdma.c
index 11a972c..25d668b 100644
--- a/migration-rdma.c
+++ b/migration-rdma.c
@@ -2507,7 +2507,6 @@ static void *qemu_rdma_data_init(const char *host_port, Error **errp)
 
     if (host_port) {
         rdma = g_malloc0(sizeof(RDMAContext));
-        memset(rdma, 0, sizeof(RDMAContext));
         rdma->current_index = -1;
         rdma->current_chunk = -1;
 
-- 
1.8.3.1

